routes/api/v1/forecast.js

```diff
@@ -6,15 +6,19 @@ var request = require('request');
 var api_call = request('https://api.nasa.gov/planetary/apod?api_key=DEMO_KEY', { json: true }, (err, res, body) => {
     if (err) {
       return err;
-    }else{
-      return [res.statusCode,body]
     }
+    res = [res.statusCode,body]
```

Comment:
You'll want to set this to a variable, I believe, and something that isn't named res